app/containers/App.module.scss

```diff
@@ -15,3 +15,20 @@
   display: flex;
   flex-direction: row;
 }
+
+.externalLinks {
+  display: flex;
```

Comment:
@oluwakemi-dada  I'd suggest to keep persistent approach to sort all css definitions alphabetically

Reply:
Noted